TypeCobol/Compiler/Nodes/Data.cs

```diff
@@ -760,10 +760,6 @@ public bool IsDataDescriptionGroup
             }
         }
 
-        public override bool VisitNode(IASTVisitor astVisitor)
-        {
-            return base.VisitNode(astVisitor) && astVisitor.Visit(this);
-        }
```

Comment:
No, we need to keep the visitor pattern for `DataDescription` objects. Why remove that ?
The computed property `IsDataDescriptionGroup` above is not used: either remove it or use it in your checker as it looks like related to this issue.

Reply:
Code is restored